src/components/Course.css

```diff
@@ -0,0 +1,16 @@
+div#Course {
+  aside {
+    margin-bottom: 1em;
+  }
+
+  section {
+    margin-top: 1em;
+  }
+
+  aside,
+  section {
+    border-radius: 10px;
+    border: solid 1px var(--colour-primary-1);
+    padding: 10px;
+  }
+}
```

Comment:
Ensured consistent spacing and alignment for better readability.
Maybe it's better to use prettier or other tools.

Reply:
The development team decided to use Prettier with a print-width of 96 characters and no trailing commas.  We don't have much of a coding style guide beyond that.  We never codified this, so I'll document it in the near future.

CSS modules are something that I've only recently learned about.  They make sense, so I'll try to change the CSS files accordingly.

Responsive design is Gunchin's domain.  At the moment, we just have a visual design for desktop to go by -- we shouldn't try to code for mobile and/or tablet devices until we have visual designs for them as well.